lib/bootsnap/load_path_cache/change_observer.rb

```diff
@@ -25,6 +25,12 @@ def concat(entries)
           super
         end
 
+        # uniq! keeps the first occurance of each path, otherwise preserving
+        # order, preserving the effective load path
+        def uniq!(&block)
```

Comment:
To be safer better to just define the method as `uniq(*)` and just call `super` without passing anything. This way if future versions of ruby change the method signature we don't need to change this one.

Reply:
I realized that what I have is probably incorrect if `uniq!` is passed a block (It should reinitialize in that case).